add note on `brew bundle --file=path` to README.md (#415)
Homebrew/homebrew-bundle: README.md

If a dependency is already installed and there is an upgrade available it will be upgraded.

> Note: `brew bundle` will look for the Brewfile on the directory in which it's run. To specify a path use `brew bundle --file=path`

Reviewer: ```suggestion
`brew bundle` will look for `Brewfile` in the current directory. Use `--file` to specify a path to a different `Brewfile`.
```
